src/RoslynInsertionTool/RoslynInsertionTool/VersionsUpdater.cs

```diff
@@ -169,24 +188,67 @@ private void UpdateAssemblyVersionsFile(string assemblyName, Version newVersion)
                     break;
             }
 
-            var i = content.IndexOf(variableName);
-            if (i < 0)
+            // the structure of the file is:
+            //   header: opening tag:    <#+
+            //           comments:           //
+            //   values:                     const string AssemblyNameVersion = "<version-number>";
+            //   footer: closing tag:    #>
+            var lines = content.Split('\n').Select(line => line.TrimEnd('\r')).ToList();
```

Comment:
I did this on purpose on the off chance that the local git install is ever updated to check out as `\n` instead of `\r\n`; this way it always works (TM).